src/main/groovy/com/pszymczyk/consul/ConsulProcess.groovy

```diff
@@ -23,6 +23,8 @@ class ConsulProcess implements AutoCloseable {
         this.address = address
         this.process = process
         this.simpleConsulClient = new SimpleConsulClient(address, httpPort)
+
+        addShutdownHook { this.process.destroyForcibly()}
```

Comment:
@pszymczyk regarding this one - a lot of my colleges complain that consul remains after tests(usually when tests are interupted) and then they have to manually kill it.

Reply:
I saw the same today and I propose some different approach but after some thoughts I think it is better approach then I suggested in https://github.com/pszymczyk/embedded-consul/pull/77

Reply:
I think https://github.com/flapdoodle-oss/de.flapdoodle.embed.process also use shutdown hooks.

Actually I think it would be nice to use that project as a base - we use their embedded mongodb library - works like a charm(almost :) )